### ISSUE TITLE
README/primer on the bio-domain

### ISSUE THREAD

**Issue by artclarke** [HazyResearch/snorkel#419]
The snorkel tutorial is awesome, but can be made even more accessible by writing up a short-summary of the disease domain (i.e. flesh out a little more what the data sets are, with examples of disease mentions that are true and false, and giving a dictionary definition for all the different look-up files in the data-dictionary).

To truly understand what's going on with Data Programming (tutorial 4), I had to read through a lot of the data files, and going to the http://www.biocreative.org/about/background/description/ page didn't really help that much to understand the domain.

In other words, if you are NOT an expert in the disease domain, give people a brief training example on it, and tell them what the data-sets actually mean.

**Comment by ajratner** [comment]
Hi @artclarke ,

I completely agree, and think this is a great suggestion.  In addition to doing this, we're also planning to (a) release a new non-bio-domain tutorial that should be more broadly understandable, and (b) we'll both add some description to this tutorial + a link to a related paper being submitted soon which contains further relevant background information.

Thanks for the great suggestions!
-Alex

**Comment by stephenbach** [comment]
We've added a more accessible tutorial, porting the spouse relation tutorial from DeepDive: https://github.com/HazyResearch/snorkel/tree/master/tutorials/intro

Also added some more background information and an example to the the disease tagging tutorial's README: https://github.com/HazyResearch/snorkel/blob/master/tutorials/disease_tagging/README.md
